Doc/c-api/tuple.rst

```diff
@@ -37,6 +37,19 @@ Tuple Objects
    or ``NULL`` with an exception set on failure.
 
 
+.. c:function:: PyObject* PyTuple_FromArray(PyObject *const *array, Py_ssize_t size)
+
+   Create a tuple of *size* items and copy references from *array* to the new
+   tuple.
```

Comment:
I would prefer to formulate this in words more similar to `PyTuple_New()` or `PyTuple_Pack()` ("Return a new tuple object of size ... "), but this is not so important.